src/document/generate_doc.py

```
@@ -0,0 +1,151 @@
+
+
+def generate_doc(file_path: Path, provider):
+    """
+    Document a single file and write the output to a file with suffix.
+    """
+    suffix = config.documentation_suffix
+
+    global done, current_file
+    done = False
+    current_file = file_path
+    input_path = config.input_path
```

Ruff

Local variable `input_path` is assigned to but never used

Remove assignment to unused variable `input_path`

(F841)